api/lib/domain/usecases/get-user-certification-eligibility.js

```diff
@@ -30,20 +30,21 @@ import { CertificationEligibility } from '../../../src/shared/domain/read-models
  * @param {CertificationBadgesService} params.certificationBadgesService
  * @param {ComplementaryCertificationCourseRepository} params.complementaryCertificationCourseRepository
  * @param {TargetProfileHistoryRepository} params.targetProfileHistoryRepository
- * @param {PlacementProfileService} params.placementProfileService
+ * @param {UserEligibilityService} params.userEligibilityService
  *
  * @returns {CertificationEligibility}
  */
 const getUserCertificationEligibility = async function ({
   userId,
   limitDate = new Date(),
-  placementProfileService,
+  userEligibilityService,
   certificationBadgesService,
   complementaryCertificationCourseRepository,
   targetProfileHistoryRepository,
 }) {
-  const placementProfile = await placementProfileService.getPlacementProfile({ userId, limitDate });
-  const pixCertificationEligible = placementProfile.isCertifiable();
+  const userEligibilityList = await userEligibilityService.getUserEligibilityList({ userId, limitDate });
+  const coreEligibility = userEligibilityList.coreEligibilityV2;
+  const pixCertificationEligible = coreEligibility.isCertifiable;
```

Comment:
`coreEligibility` peut être égale à `null` à en croire le getter `coreEligibilityV2`.

Cette ligne peut générer une erreur.

```suggestion
const pixCertificationEligible = coreEligibility?.isCertifiable;
```

Reply:
+1 pour le getter qui peut retourner un null. 
Après le `UserEligibilityCalculator` semble push systématiquement des core.

``` 
this.#eligibilities.push(buildCoreEligibility({ isCertifiable }));
    this.#eligibilitiesV2.push(buildCoreEligibility({ isCertifiable }));
```

Reply:
Est-ce que cela pourrait etre simplifie avec une methode dans l'Aggregate qui ferait le travail de ces deux lignes pour renvoyer un boolean ?

```suggestion
  const pixCertificationEligible = userEligibilityList.isEligibleToCore();
```